### PR TITLE
[SW-1487] Update examples/README file

### PR DESCRIPTION
**Pull request by jakubhava**
Fix links on the examples/Readme file to point to built documentation.

Also when doing the README update, updated quickly the example script so it can run again

### CODE REVIEW

File: examples/README.rst [hunk 1]
```diff
@@ -85,83 +87,60 @@ Step-by-Step Weather Data Example
...
-    from the RDD API:
+4.  Load weather data for Chicago international airport (ORD):
 
 .. code:: scala
 
-    import org.apache.spark.examples.h2o._
     val weatherDataFile = "examples/smalldata/chicago/Chicago_Ohare_International_Airport.csv"
-    val wrawdata = spark.sparkContext.textFile(weatherDataFile,3).cache()
-    val weatherTable = wrawdata.map(_.split(",")).map(row => WeatherParse(row)).filter(!_.isWrongRow())
+    val weatherTable = spark.read.option("header", "true").option("inferSchema", "true").csv(weatherDataFile).withColumn("Date", to_date('Date, "MM/dd/yyyy")).withColumn("Year", year('Date)).withColumn("Month", month('Date)).withColumn("DayofMonth", dayofmonth('Date))
```
Comment (mn-mikke) [inline examples/README.rst]: WDYT about referomating the logic into multiple lines. (via the `select` statement or user can use `:paste` mode.

File: examples/README.rst [hunk 2]
```diff
@@ -85,83 +87,60 @@ Step-by-Step Weather Data Example
...
+    val flightsToORD = airlinesTable.filter('Dest === "ORD")
 
 7.  Compute the number of these flights:
 
 .. code:: scala
 
     flightsToORD.count
 
-8.  Use Spark SQL to join the flight data with the weather data:
+8.  Join the flight and weather data frames:
```
Comment (mn-mikke) [inline examples/README.rst]: `flights`?
just `weather data frame` or `weather data`?

File: examples/README.rst (hunk 1 above)
Comment (jakubhava) [reply examples/README.rst]: 👍

File: examples/README.rst (hunk 2 above)
Comment (jakubhava) [reply examples/README.rst]: what do you mean pls?

Comment (mn-mikke) [reply examples/README.rst]: Here we a dataset with flights and join it with weather data, right?

Comment (jakubhava) [reply examples/README.rst]: reworded

**Pull request merged**